### DISCUSSION TITLE
HW 1 Part 2 Q1

### DISCUSSION THREAD

**Discussion by caroline-cook** [cooperrc/computational-mechanics#27] [Q&A]
I was wondering what part a meant, to document the function with a help file in '''help'''? Does this just mean to add explanation to your code as you go? Thanks!

**Comment by FishMacaron** [comment]
For functions in particular you can add a commented out section that will come up when called for in your code. For example:

> def functionname():
    '''
    THIS IS WHERE YOU WRITE ABOUT WHAT YOUR FUNCTION DOES SO IT CAN BE CALLED LATER
    '''
    print("the rest of your code continues here as normal")

You can then call the help file in your code with: help(*your function name*)

If you need more clarification or further help I think reading the "Commenting your code" section of the Getting Started section may help! But I'm willing to help more if you still need it, otherwise I hope this helped a bit!
